### DISCUSSION TITLE
Alarm

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk-server-pro#77] [Q&A]
Can someone explain to me when an alarm is entered in the log? Also when an email is sent if the user has the check on send email on alarm.

Thanks 1000

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/56#issuecomment-1689879026 again

**Reply by pdifeo** [reply]
I already understood that it will be present in the next version (when?).. The question was a different one. I would like to know how it works, if possible.  Otherwise I will wait and understand in due course.

**Reply by rustdesk** [reply]
@dinger1986 We owe our users a lot of documentation, and apparently no one in our team is competent on this the job right now. The documentation is written very bad, no matter the part I have written and you have written.

**Reply by rustdesk** [reply]
Will release next version tommorrow or so.

**Reply by dinger1986** [reply]
> @dinger1986 We owe our users a lot of documentation, and apparently no one in our team is competent on this the job right now. The documentation is written very bad, no matter the part I have written and you have written.

Yes I agree there's a lot of gaps in the documentation which need filled

**Reply by pdifeo** [reply]
when will the next version come out?

**Comment by pdifeo** [comment]
Despite having installed version 1.1.10 released today, I still don't understand when an alarm is 'triggered'.

I have tried from the login client with incorrect credentials 50 times, and got nothing . Neither alarms nor the blocking of the client trying to log in.

https://github.com/rustdesk/rustdesk-server-pro/assets/8618744/34b29c3c-ab32-4f94-a9b2-d715375a2332

<img width="648" alt="image" src="https://github.com/rustdesk/rustdesk-server-pro/assets/8618744/b20b6680-1c1d-49ab-84a7-eaf1f9fb168e">

**Reply by pdifeo** [reply]
The alarm log is present only when try to access to peer ... And in any case id of the peer that try to connect is not showed, only the IP . This is not useful

<img width="922" alt="image" src="https://github.com/rustdesk/rustdesk-server-pro/assets/8618744/6ce15279-060e-42a7-8000-793de78d0fdf">

**Comment by 21pages** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/70#issuecomment-1732684183
